Projects/Server/Items/Item.cs

```diff
@@ -599,18 +599,16 @@ public IEntity Parent
 
             var oldParent = m_Parent;
 
+            if (m_Map != null && oldParent == null && value != null)
+            {
+                m_Map.OnLeave(this);
+            }
+
             m_Parent = value;
 
-            if (m_Map != null)
+            if (m_Map != null && oldParent != null && value == null)
             {
-                if (oldParent != null && m_Parent == null)
-                {
-                    m_Map.OnEnter(this);
-                }
-                else if (m_Parent != null)
-                {
-                    m_Map.OnLeave(this);
-                }
+                m_Map.OnEnter(this);
```

Comment:
When an item is lifted, it is effected moved to the Map's Sector0 list, even though it's location is never set to 0,0,0.
TODO: Tighten up the movement logistics. This shouldn't be allowed.